CONTRIBUTING.md

```diff
@@ -3,52 +3,45 @@ Contributing to Paho
 
 Thanks for your interest in this project.
 
-Project description:
+Project Description:
```

Comment:
Here is the full suggested contributor agreement from the paho project (I think most of these points are covered and having this customised to the project is good). It would be good to copy the full section re the contributor agreement (as I think it's important to explain why this is required) and I guess the terms of use section should be there (legal team probably would like that!).

# Contributing to Eclipse Paho

Thanks for your interest in this project.

## Project description

The Eclipse Paho project provides reliable open-source implementations of open
and standard messaging protocols aimed at new, existing, and emerging
applications for Machine-to-Machine (M2M) and Internet of Things (IoT). Paho
reflects the inherent physical and cost constraints of device connectivity. Its
objectives include effective levels of decoupling between devices and
applications, designed to keep markets open and encourage the rapid growth of
scalable Web and Enterprise middleware and applications. Paho contains MQTT
publish/subscribe client implementations for use on embedded platforms, along
with corresponding server support as determined by the community.

* https://projects.eclipse.org/projects/iot.paho

## Terms of Use

This repository is subject to the Terms of Use of the Eclipse Foundation

* https://www.eclipse.org/legal/termsofuse.php

## Developer resources

Information regarding source code management, builds, coding standards, and
more.

* https://projects.eclipse.org/projects/iot.paho/developer

The project maintains the following source code repositories

* https://github.com/eclipse/paho-website
* https://github.com/eclipse/paho.golang
* https://github.com/eclipse/paho.mqtt-sn.embedded-c
* https://github.com/eclipse/paho.mqtt-spy
* https://github.com/eclipse/paho.mqtt.android
* https://github.com/eclipse/paho.mqtt.c
* https://github.com/eclipse/paho.mqtt.cpp
* https://github.com/eclipse/paho.mqtt.d
* https://github.com/eclipse/paho.mqtt.embedded-c
* https://github.com/eclipse/paho.mqtt.golang
* https://github.com/eclipse/paho.mqtt.java
* https://github.com/eclipse/paho.mqtt.javascript
* https://github.com/eclipse/paho.mqtt.m2mqtt
* https://github.com/eclipse/paho.mqtt.python
* https://github.com/eclipse/paho.mqtt.ruby
* https://github.com/eclipse/paho.mqtt.rust
* https://github.com/eclipse/paho.mqtt.testing

## Eclipse Development Process

This Eclipse Foundation open project is governed by the Eclipse Foundation
Development Process and operates under the terms of the Eclipse IP Policy.

* https://eclipse.org/projects/dev_process
* https://www.eclipse.org/org/documents/Eclipse_IP_Policy.pdf

## Eclipse Contributor Agreement

In order to be able to contribute to Eclipse Foundation projects you must
electronically sign the Eclipse Contributor Agreement (ECA).

* https://www.eclipse.org/legal/ECA.php

The ECA provides the Eclipse Foundation with a permanent record that you agree
that each of your contributions will comply with the commitments documented in
the Developer Certificate of Origin (DCO). Having an ECA on file associated with
the email address matching the "Author" field of your contribution's Git commits
fulfills the DCO's requirement that you sign-off on your contributions.

For more information, please see the Eclipse Committer Handbook:
https://www.eclipse.org/projects/handbook/#resources-commit

## Contact

Contact the project developers via the project's "dev" list.

* https://dev.eclipse.org/mailman/listinfo/paho-dev

Reply:
@MattBrittan Thank you for the resource and suggestions!
I made some change based on them.